docs/sdkx-xamarin/migration-guide.mdx

```diff
@@ -0,0 +1,527 @@
+| -------------------- | ------------------------------------ | ----------------------------------- |
+| Logout               | `HelpshiftCore.Logout()`             | `Helpshift.Logout()`                |
+| Clear anonymous user | `HelpshiftCore.ClearAnonymousUser()` | `Helpshift.ClearAnonymousUser(bool)`|
+
+For more information about User management, check the [Users](/sdkx-xamarin/users/) guide.
+
+#### Design and Theming
+
+SDK’s design and theming have been completely moved to the admin dashboard in SDK X. Please remove all calls to theming APIs and all theming-related files from your project. Removed APIs -
+
+- `SetThemes(lightThemeFileName, darkThemeFileName)`
+
+- `SetTheme(themeName)`
+
+Theming can be configured from the admin dashboard. You can find more details in the [SDK X theming](https://support.helpshift.com/kb/article/configure-look-and-feel-settings/) guide.
+
+#### String Customization
+
+String customization is yet to be supported. It will move to Helpshift Dashboard. Remove all Helpshift SDK related string customizations done in your project as they will no longer reflect in SDK X.
+
+#### Notification APIs
+
+The API signatures have changed. Update existing API calls with new ones as mentioned -
+
+| Action                       | Legacy SDK Xamarin plugin API                       | SDK X Xamarin plugin API                           |
+| ---------------------------- | -------------------------------------------------   | -------------------------------------------------- |
+| Register push token          | `HelpshiftCore.RegisterDeviceToken(string)`         | `Helpshift.RegisterPushToken(string)`              |
+| Request unread message count | `HelpshiftSupport.RequestUnreadMessagesCount(bool)` | `Helpshift.RequestUnreadMessageCount(bool)`        |
+
+The delegate method that receives the unread message count has also changed from -
+
+```csharp
+// Define Helpshift support delegate
+public class  HSDelegate : ISupportDelegate 
+{
+    public void DidReceiveUnreadMessagesCount(int count)
+    {
+      // Legacy SDK unread message count handling here
+    }
+}
+
+// Register delegate with Helpshift
+HSDelegate delegateObject = new HSDelegate();
+HelpshiftSupport.SetDelegate(delegateObject);
+
+// Request unread message count
+HelpshiftSupport.RequestUnreadMessagesCount(true);
+```
+
+to -
+
+```csharp
+// Define Helpshift events listener
+public class  HSDelegate : IHelpshiftEventsListener 
+{
+  public void OnEventOccurred(string eventName, IDictionary<string, object> data)
+  {
+    switch(eventName)
+    {
+      case HelpshiftEvent.RECEIVED_UNREAD_MESSAGE_COUNT:
+        // SDK X unread message count handling here
+        break;
+    }
+  }
+}
+
+// Register events listener with Helpshift
+Helpshift.SetHelpshiftEventsListener(new HSDelegate());
+Helpshift.SetHelpshiftEventsListener(this);
+
+// Request unread message count
+Helpshift.RequestUnreadMessageCount(true);
```

Comment:
We should provide link to the SDK X Delegates section here sonewhere for reference.